README.md

```diff
@@ -46,21 +45,32 @@ python3 install.py <flavor> <accent>
   [catppuccin-gtk] [INFO] - Theme installation complete! 
 ```
 
+### Flatpak
+In order to give flatpaks access to the theme directory, run this command:
+```bash
+sudo flatpak override --filesystem=$HOME/.local/share/themes
+```
+
+Then, run this command to set the theme:
+```bash
+export THEME_DIR="~/.local/share/themes/catppuccin-<flavor>-<accent>-standard+default"
+sudo flatpak override --env=GTK_THEME=$THEME_DIR
+```
+
```

Comment:
Flatpak was removed from the README intentionally. We no longer support this avenue of installation. We do not maintain the Flatpak and it has been left with periods of inactivity more than once.
```suggestion
```

Reply:
oh